### PR TITLE
removing the slice option for FF

### PR DESCRIPTION
**Pull request by ragusa**

### CODE REVIEW

File: framework/mesh/mesh_continuum/mesh_continuum.cc [hunk 1]
```diff
@@ -62,9 +62,9 @@ MeshContinuum::MakeMPILocalCommunicatorSet() const
   {
     int locI_num_connections = static_cast<int>(local_connections.size());
 
-    // If chi::mpi.location_id == locI then this call will
+    // If mpi.location_id == locI then this call will
     // act like a send instead of receive. Otherwise
-    // It receives the count.
+    // it receives the count.
     mpi_comm.broadcast(locI_num_connections, locI);
```
Comment (andrsd) [inline framework/mesh/mesh_continuum/mesh_continuum.cc:68]: I would maybe even remove this comment. This is what broadcast does, so no need to explain how MPI works.

File: framework/mesh/mesh_continuum/mesh_continuum.cc [hunk 2]
```diff
@@ -81,9 +81,9 @@ MeshContinuum::MakeMPILocalCommunicatorSet() const
   // Broadcast local connections
   for (int locI = 0; locI < opensn::mpi_comm.size(); locI++)
   {
-    // If chi::mpi.location_id == locI then this call will
+    // If mpi.location_id == locI then this call will
     // act like a send instead of receive. Otherwise
-    // It receives the count.
+    // it receives the count.
     mpi_comm.broadcast(global_graph[locI].data(), global_graph[locI].size(), locI);
```
Comment (andrsd) [inline framework/mesh/mesh_continuum/mesh_continuum.cc:87]: The same thing here...

**Review by zhardy-lanl — APPROVED** [review]
I concur with @andrsd comments.

File: framework/mesh/mesh_continuum/mesh_continuum.cc (hunk 1 above)
Comment (ragusa) [reply framework/mesh/mesh_continuum/mesh_continuum.cc:64]: done

File: framework/mesh/mesh_continuum/mesh_continuum.cc (hunk 2 above)
Comment (ragusa) [reply framework/mesh/mesh_continuum/mesh_continuum.cc:75]: done

**Review by andrsd — APPROVED** [review]

**Review by wdhawkins — APPROVED** [review]

**Pull request merged**